src/SparkFun_SCD30_Arduino_Library.cpp

```diff
@@ -50,14 +50,27 @@ bool SCD30::begin(TwoWire &wirePort, bool autoCalibrate)
    *
    * To set ClockStretchlimit() a check for ESP8266 boards has been added in the driver.
    *
-   * With setting to 20000, we set a max timeout of 20mS (> 20x the maximum measured) basically disabling the time-out 
+   * With setting to 20000, we set a max timeout of 20mS (> 20x the maximum measured) basically disabling the time-out
    * and now wait for clock stretch to be controlled by the client.
    */
 
 #if defined(ARDUINO_ARCH_ESP8266)
   _i2cPort->setClockStretchLimit(200000);
 #endif
 
+  uint16_t fwVer;
+  if (getFirmwareVersion(&fwVer) == false) // Read the firmware version. Return false if the CRC check fails.
+    return (false);
+
+  if (_printDebug == true)
+  {
+    _debugPort->print("SCD30 begin: got firmware version 0x");
```

Comment:
Do you want a F() around these to force to flash?

Reply:
Oh, go on then... Twist my arm...
:-D

Reply:
Resolved [here](https://github.com/sparkfun/SparkFun_SCD30_Arduino_Library/commit/6d98e8dd9fd7a3ed62f4ca59217b94199d7b2da7)